Makefile

```diff
@@ -0,0 +1,43 @@
+TARGETS ?= darwin/amd64 linux/amd64 windows/amd64
+DIST_DIRS = find * -type d -exec
+
+# go options
+BINDIR := $(CURDIR)/bin
+GO ?= go
+GOFLAGS :=
+TAGS :=
+LDFLAGS := -w -s
+
+.PHONY: build
+build:
+	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' github.com/deislabs/smi-sdk-go/cmd/installer
+
+.PHONY: build-cross
+build-cross: LDFLAGS += -extldflags "-static"
+build-cross:
+	CGO_ENABLED=0 gox -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/{{.Dir}}" -osarch='$(TARGETS)' $(GOFLAGS) $(if $(TAGS),-tags '$(TAGS)',) -ldflags '$(LDFLAGS)' github.com/deislabs/smi-sdk-go/cmd/installer
+
+.PHONY: checksum
+checksum:
+	for f in _dist/*.{gz,zip} ; do \
+			shasum -a 256 "$${f}"  | awk '{print $$1}' > "$${f}.sha256" ; \
+			echo -n "Checksum: " && cat $${f}.sha256 ; \
+	done
+
+.PHONY: dist
+dist:
+	( \
```

Comment:
Why is this a subshell?

Reply:
Needs to be otherwise the bash script won't work correctly. Did you notice that DIST_DIRS gets calculated with the `find` command in [this line](https://github.com/deislabs/smi-sdk-go/pull/23/files#diff-b67911656ef5d18c4ae36cb6741b7965R2)? Dunno if that makes it clearer. I could also move this script to its own file and run it from this make target but that seems unnecessary. Open to suggestions!